test/unit/fixtures/config/timeout/config/default.json

```diff
@@ -0,0 +1,9 @@
+{
+"express": {
+    "httpServer": {
```

Comment:
per the previous comment, I think the best would be to have a single key that applies to the right Node.js key based on the node version...

Reply:
in which case, the config probably wouldn't go with `express.httpServer` but somewhere else?

separately, but related ... generally, should we be able to add anything to `express.httpServer` and have it pass to the `server` config?

Reply:
Perhaps I'm misunderstanding your point, but all of these keys could apply to the same node version, at least if you are using node v11.3.0 or later, which supports configuring all three http timeout related settings.  Each of these timeout settings configures a unique node http server setting, and have been introduced progressively in the 8.0 and 11.3 releases of node.

I was also trying to make my changes consistent with the existing express / node configuration code where even though express is managing the creation of the node http server for non-ssl connections and its using the node https object to create the server and establish the listening port for ssl connections, the config object simply models the port setting using a consistent express.port setting. If the node httpServer settings were modelled as a separate config attribute, would it make sense for ssl connections that are in-fact instantiated with node's http object not express?  

I also considered an implementation similar to what you are suggested, one that would simply pass values from the config object to the node server, by parsing the httpServer object and then passing the key and value entries to the node.server object, this also seemed like more complexity than necessary as new node  http server settings are not introduced with much frequency, there have only been a few new additions over the release history of node, though I do like the concept of the BlueOak code being more agnostic to the specifics of the node setting schema.

If you are open to continued incremental refinement, perhaps I could implement that improvement in a future PR.   Implementing that change later would not affect any existing exploiters of this current implementation, given the current express.httpServer config object structure is maintained.

Reply:
ok, first, I misunderstood and thought that Node.js was changing the config key across their releases!
I think the simple solution works at present.

Reply:
also, additional PRs are always welcome :-D

Reply:
Thanks for the feedback Sean, I do like the idea, I'll work on a solution that handles future node http-server setting introductions without the need to change the BO code and submit it as a separate PR.